### ISSUE TITLE
broad body size causes shirt to display over armor when not using costume

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#10473]
If you set up your avatar like this:
- costume turned off (i.e., avatar displays Battle Gear)
- armor piece selected in Battle Gear
- any shirt colour selected
- `broad` body size chosen

then your avatar displays your shirt, not your armor.

It does not occur if you change the body size to `slim` or if you have the costume option turned on.

Example:
```
{
    "_id": "2f08bb78-dd49-42db-9182-71e7d72b9a99",
    "profile": {
        "name": "Nevaline"
    },
    "items": {
        "gear": {
            "owned": {
                ...,
                "weapon_special_summer2018Mage": true,
                "armor_special_summer2018Mage": true,
                "head_special_summer2018Mage": true
            }
            "equipped": {
                "armor": "armor_special_summer2018Mage",
                "head": "head_special_summer2018Mage",
                "shield": "shield_base_0",
                "eyewear": "eyewear_base_0",
                "weapon": "weapon_special_summer2018Mage",
                "headAccessory": "headAccessory_base_0"
            }
            "costume": {
                "armor": "armor_base_0",
                "head": "head_base_0",
                "shield": "shield_base_0"
            }
        }
    },
    "preferences": {
        "size": "broad",
        "shirt": "blue",
        "costume": false,
        ...
    },
}
```
![image](https://user-images.githubusercontent.com/1495809/41814324-1cc7a678-778c-11e8-85bf-9d8906113435.png)

**Comment by Alys** [comment]
Note that I haven't checked my assumption that it's a z-index problem. Might not be.

**Comment by Alys** [comment]
The brilliant and awesome  citrusella, the serving wench has worked out what's causing this:

"@Nevaline I don't actually do the coding or art or anything, but snooping through the code with my rather minimal knowledge of how it works, it appears the broad version of the armor you're wearing may mistakenly be a copy of the hand item. (I'm using [this file](https://github.com/HabitRPG/habitica/blob/develop/website/raw_sprites/spritesmith/gear/events/summer/broad_armor_special_summer2018Mage.png) (and [this](https://github.com/HabitRPG/habitica/blob/8437b916c4a41f655ee6c56ac30bd58a0082b642/website/client/assets/images/sprites/spritesmith-main-8.png) and [this](https://github.com/HabitRPG/habitica/blob/8437b916c4a41f655ee6c56ac30bd58a0082b642/website/client/assets/css/sprites/spritesmith-main-8.css#L1105), though I'm a little brain foggy and haven't confirmed that's what they say) to come to that conclusion, as a reference for anyone who knows what's up.)"_

**Comment by SabreCat** [comment]
oh no! What a goof. This is almost certainly my error. I'll rebuild these sprites.

**Comment by SabreCat** [comment]
Yup yup. We attempted to fix a transparency issue with the weapon, and the fixed file had the wrong name...

**Comment by SabreCat** [comment]
Fixed in bc03c1d18a9738d146e920f51adf31be671a0d56
